package.json

```diff
@@ -81,7 +81,7 @@
     "semantic-release": "^19.0.2",
     "semantic-release-monorepo": "^7.0.5",
     "ts-jest": "^24.1.0",
-    "typescript": "4.3.5"
+    "typescript": "^4.9.3"
```

Comment:
Some types are not supported under the locked 4.3.5 version, such as `window.ClipboardItem` and `navigator.clipboard.write`.

And I kept all the typescript versions consistent in the repository.

Reply:
okay (lock version was due to an error